sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/api/Azure.ResourceManager.RedisEnterprise.netstandard2.0.cs

```diff
@@ -159,11 +165,52 @@ public partial class RedisEnterprisePrivateEndpointConnectionResource : Azure.Re
 }
 namespace Azure.ResourceManager.RedisEnterprise.Models
 {
+    public partial class Capability
```

Comment:
Single word class name is not allowed, please specify its name.
Simply `RedisEnterpriseCapability` should work if nothing better comes to your mind

Reply:
updated